packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -362,6 +362,7 @@ const ContractTypeGlossary = ({
                             key={text.props.i18n_default_text}
                             weight='bold'
                             className='contract-type-info__content-glossary--heading'
+                            data-glossary-term={text.props.i18n_default_text.toLowerCase().replace(/\s+/g, '_')}
```

Comment:
is relying on text.props.i18n_default_text a good idea ? or just add one more property in the `content` and use it that for this purpose